### PR TITLE
SECURITY.md

### PR DESCRIPTION
**Pull request by auttakorngit**
### Pull request details

- [x] This PR is a bug fix
- [x] This PR implements a new feature or introduces new behavior.

### CODE REVIEW

**Review by auttakorngit — COMMENTED** [review]
SECURITY

**Comment by the-j0k3r** [comment]
@auttakorngit Thanks for this PR.

I dont understand the versions you have added, none match our release versions.
further if you read #543 this is a community driven project and were there any security issues, who would fix them?

This project needs developers. And the project owner wants to archive the project so the project needs a new home and developers. Did I mention we need developers?

Thanks for understanding.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.1.x   | :white_check_mark: |
+| 5.0.x   | :x:                |
+| 4.0.x   | :white_check_mark: |
+| < 4.0   | :x:                |
```
Comment (the-j0k3r) [inline SECURITY.md:13]: This seems to be the default security file GitHub creates, I dont think this is even suitable for our project

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Reporting a Vulnerability
+
+Use this section to tell people how to report a vulnerability.
+
+Tell them where to go, how often they can expect to get an update on a
```
Comment (the-j0k3r) [inline SECURITY.md:19]: Yes this is a problem, theres no active devs that have shown interest or said that are qualified to handle these reports, never mind fix them.

Meanwhile the vacancy is open for a volunteer, if you or someone you know can audit our code and fix it I would be happy to talk about it further.

**Comment by the-j0k3r** [comment]
Ill reopen this to give other people the opportunity to chime in. Shouldn't have closed it.

**Comment by the-j0k3r** [comment]
@auttakorngit did you wish to improve this PR or no?

**Pull request closed without merging**